src/storage/src/hummock/sstable/block_iterator.rs

```diff
@@ -77,7 +76,8 @@ impl BlockIterator {
 
     pub fn key(&self) -> FullKey<&[u8]> {
         assert!(self.is_valid());
-        FullKey::decode(&self.key)
+
+        FullKey::from_slice_without_table_id(self.block.table_id(), &self.key[..])
```

Comment:
It seems that we concate the block_key and block_id 

how about `from_slice_with_table_id`?